src/idl_gen_cpp.cpp

```diff
@@ -278,7 +278,7 @@ class CppGenerator : public BaseGenerator {
           code_ += "bool operator==(const " + nativeName + " &lhs, const " +
                    nativeName + " &rhs);";
           code_ += "bool operator!=(const " + nativeName + " &lhs, const " +
-              nativeName + " &rhs);";
+                   nativeName + " &rhs);";
         }
```

Comment:
here the difference is only in the operator names: `bool operator==` vs `bool operator!=`.
Can we write like this?
```cpp
auto signature = "(const " + nativeName + " &lhs, const " + nativeName + " &rhs);";
code += "bool operator==" + signature;
code += ...
```
?

Reply:
I'm wrong. This code isn't part of this PR.
You just fixed the wrong code formatting.